packages/twenty-front/src/modules/views/hooks/useVectorSearchInputState.ts

```diff
@@ -0,0 +1,27 @@
+import { useRecoilComponentStateV2 } from '@/ui/utilities/state/component-state/hooks/useRecoilComponentStateV2';
+import { vectorSearchInputComponentState } from '@/views/states/vectorSearchInputComponentState';
+import { isDefined } from 'twenty-shared/utils';
+import { useVectorSearchFilterOperations } from './useVectorSearchFilterOperations';
+
+export const useVectorSearchInputState = (filterDropdownId: string) => {
+  const [vectorSearchInputValue, setVectorSearchInputValue] =
```

Comment:
This doesn't have to be exported from a hook, it can be used as is where needed, no need to wrap the useRecoilComponentStateV2 calls.